src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -1,18 +1,34 @@
 package frc.robot.subsystems.intake;
 
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.RobotConstants.IntakeConstants;
 
 public class Intake extends SubsystemBase {
-  /** Creates a new ExampleSubsystem. */
-  public Intake() {}
+  private final CANSparkMax m_rollerMotor; // Intake roller motor
+
+  // Constructs intake and initializes motor
+  public Intake() {
+    m_rollerMotor = new CANSparkMax(IntakeConstants.kMotorID, MotorType.kBrushless);
```

Comment:
Just a note: depending on how the motor is mounted you may need to set its "inverted" state to `true`. No action necessary at this time.